index.html

```diff
@@ -14,8 +14,98 @@
 				<li><a href="#">Contact</a></li>
 			</ul>
 		</nav>
-		
-
 	</div>
+	<section class="F-section"><p class="T-name">Amin Dream Team</p>
+		<img src="https://img.icons8.com/metro/208/000000/avengers.png">
+	</section>
+	<section class="sec2">
+		<div class="team">
+			<p class="t-mem" align="center">Team Members</p>
+			<div class="card">
+				<div class="m-img">
+				</div>
+				<div class="m-desc">
+					<h2>Amin King</h2>
+					<p class="title">ENG &amp; DEV</p>
+					<p>Some text that describes me lorem ipsum ipsum lorem.</p>
+					<p>example@example.com</p>
+					<p><button class="button">Contact</button></p>
+				</div>
+			</div>
+			<div class="card">
+				<div class="m-img">
+				</div>
+				<div class="m-desc">
+					<h2>Amin King</h2>
+					<p class="title">ENG &amp; DEV</p>
+					<p>Some text that describes me lorem ipsum ipsum lorem.</p>
+					<p>example@example.com</p>
+					<p><button class="button">Contact</button></p>					
+				</div>
+			</div>
+			<div class="card">
+				<div class="m-img">
+				</div>
+				<div class="m-desc">
+					<h2>Amin King</h2>
+					<p class="title">ENG &amp; DEV</p>
+					<p>Some text that describes me lorem ipsum ipsum lorem.</p>
+					<p>example@example.com</p>
+					<p><button class="button">Contact</button></p>					
+				</div>
+			</div>
+			<div class="card">
+				<div class="m-img last">
+				</div>
+				<div class="m-desc">
+					<h2>Amin King</h2>
+					<p class="title">ENG &amp; DEV</p>
+					<p>Some text that describes me lorem ipsum ipsum lorem.</p>
+					<p>example@example.com</p>
+					<p><button class="button">Contact</button></p>					
+				</div>
+			</div>			
+		</div>
+	</section>
+	<section class="sec3">
+			<p class="t-mem" align="center">Team Skills</p>
+		<div class="t-skill">
+		<img src="img/3.png" class="t-skill">
+		</div>
+	</section>
+
+	<section class="sec3">
+		<p class="t-mem" align="center">Contact area</p>
+		<div class="contact-area">
+		<div class="img-contact">
+			<img src="img/4.png" >
+		</div>
+		<div class="contact">
+			<form>
+			<h3>Contact Form</h3>
+			<h4>Contact us for work</h4>
+			<fieldset>
+			<input class="input" placeholder="Your name" type="text"required autofocus>
+			</fieldset>
+			<fieldset>
+			<input class="input" placeholder="Your Email Address" type="email"required>
+			</fieldset>
+			<fieldset>
+			<input class="input" placeholder="Your Phone Number (optional)" type="tel"required>
+			</fieldset>
+			<fieldset>
+			<input class="input" placeholder="Your Web Site (optional)" type="url"required>
```

Comment:
Telling the user it's optional then adding "required" to the field is bad user experience.

If it's optional then it shouldn't be required. Also, don't forget to add space between the keyword ```required``` and the quotation mark before it.
```html
"url" required
```